src/client/datascience/jupyter/kernels/kernelExecution.ts

```diff
@@ -129,13 +129,23 @@ export class KernelExecution implements IDisposable {
                     traceInfo(
                         `Cancel all remaining cells ${cancelTokenSource.token.isCancellationRequested} || ${executionResult}`
                     );
-                    await Promise.all(codeCellsToExecute.map((cell) => cell.cancel())); // Cancel pending cells.
+                    await Promise.all(
```

Comment:
Isn't the traceInfo above now off?

Reply:
No I don't think so? It's cancelling all remaining cells. What does it seem like it's doing to you?

Reply:
lol, my mistake. I thought we were changing this to try to run the cells after the one that errored, I was confused by the code and comment.